docs/commands/languages.asciidoc

```diff
@@ -0,0 +1,76 @@
+== languages
```

Comment:
This command should be rolled into part of the "info" command if the eng translation functionality gets promoted from hoot-rnd to hoot-core.